### ISSUE TITLE
binary download, golangci-lint fails to run even a simple --version.

### ISSUE THREAD

**Issue by tcanabrava** [golangci/golangci-lint#3878]
### Welcome

- [X] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [X] Yes, I've searched similar issues on GitHub and didn't find any.
- [X] Yes, I've included all information below (version, config, etc.).
- [X] Yes, I've tried with the standalone linter if available (e.g., gocritic, go vet, etc.). (https://golangci-lint.run/usage/linters/)

### Description of the problem

```
➜  accounts-manager git:(tcanabrava/accounts-manager-testing-v3) ✗ curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.53.1
golangci/golangci-lint info checking GitHub for tag 'v1.53.1'
golangci/golangci-lint info found version: 1.53.1 for v1.53.1/linux/amd64
golangci/golangci-lint info installed /home/tcanabrava/go/bin/golangci-lint

✗ /home/tcanabrava/go/bin/golangci-lint --version
ERRO [linter] depguard: create analyzer: must have an Allow and/or Deny package list 

✗ /home/tcanabrava/go/bin/golangci-lint --help
ERRO [linter] depguard: create analyzer: must have an Allow and/or Deny package list 

✗ /home/tcanabrava/go/bin/golangci-lint --run
ERRO [linter] depguard: create analyzer: must have an Allow and/or Deny package list 
```

I haven't configured depguard - but I believe that error messages should be improved to help me fix the issue locally. I also tried to search & configure depguard but having a .depguard.yaml on my source fouder didn't work too.

### Version of golangci-lint

<details>

```console
$ golangci-lint --version
# Paste output here
```

</details>

### Configuration file

<details>

```console
$ cat .golangci.yml
# paste output here
```

</details>

### Go environment

<details>

```console
$ go version && go env
# paste output here
```

</details>

### Verbose output of running

<details>

```console
$ golangci-lint cache clean
$ golangci-lint run -v
# paste output here
```

</details>

### Code example or link to a public repository

<details>

```go
// add your code here
```

</details>
